src/main/java/frc/robot/subsystems/SwerveDrive.java

```diff
@@ -401,4 +404,17 @@ public void periodic() {
                 SmartDashboard.putNumber("BLDrivePosition", m_backLeftModule.getDriveController().getMotor().getSelectedSensorPosition());
                 SmartDashboard.putNumber("BRDrivePosition", m_backRightModule.getDriveController().getMotor().getSelectedSensorPosition());
         }
+
+        @Override
+        public void drive(VelocityCommand command) {
```

Comment:
Could the ChassisInterface be changed so that it accepts a ChassisSpeeds object instead of a VelocityCommand? They contain basically the same info. Then we don't need a separate drive method....one already exists accepting a ChassisSpeeds object.